doc/source/timeseries.rst

```diff
@@ -265,6 +265,22 @@ Typical epoch stored units
    pd.to_datetime([1349720105100, 1349720105200, 1349720105300,
                    1349720105400, 1349720105500 ], unit='ms')
 
+Conversion of float epoch times can lead to inaccurate and unexpected results.
```

Comment:
i would make this a note

also units are rounded to the unit precision so the first example is misleading